src/Sidebar.ts

```diff
@@ -54,13 +57,26 @@ export class Sidebar {
         mp.enabled = !mp.enabled;
       }, mp.enabled);
       right.appendChild(toggle);
+
+      const cleanup = this.makePushButton("x", () => mp.module.cleanup());
```

Comment:
not sure why this is included in the MR?

Reply:
I wanted to test that module's cleanup works correctly. Ended up adding this because its useful to see if our code is messing up the interface in any way. Still this isn't a replacement for the settings page we were talking about.
Another addition that I'm hoping to add is a wrapper for all the dom functions that on failure to match any expected element it disables the module.